ext/dom/html5_serializer.c

```diff
@@ -357,4 +344,22 @@ zend_result dom_html5_serialize(dom_html5_serialize_context *ctx, const xmlNode
 	return dom_html5_serialize_node(ctx, node->children, node);
 }
 
+/* Variant on the above that is equivalent to the "outer HTML". */
+zend_result dom_html5_serialize_outer(dom_html5_serialize_context *ctx, const xmlNode *node)
```

Comment:
Does this need some PHP_DOM_API or whatever magic linker symbol is needed? (I don't understand why sometimes it is necessary and other times not)

Reply:
No. We only need that macro when exposing functions to other extensions.

Reply:
ACK